game-core/src/main/java/games/strategy/engine/framework/GameDataManager.java

```diff
@@ -120,6 +109,23 @@ public static GameData loadGame(final InputStream is) throws IOException {
     }
   }
 
+  private static boolean promptToLoadNewerSaveGame(final Version saveGameVersion) {
+    final int answer = Interruptibles.awaitResult(() -> SwingAction.invokeAndWaitResult(() -> {
+      final String message = "Your TripleA engine is OUT OF DATE. "
+          + "This save was made by a newer version of TripleA.\n\n"
+          + "However, because the first 3 version numbers are the same as your current version, we can still open the "
+          + "save.\n\n"
+          + "This TripleA engine is version " + ClientContext.engineVersion().toStringFull() + " and you are trying "
+          + "to open a save made with version " + saveGameVersion.toStringFull() + "\n\n"
+          + "To download the latest version of TripleA, please visit "
+          + UrlConstants.LATEST_GAME_DOWNLOAD_WEBSITE + ".\n\n"
+          + "It is recommended that you upgrade to the latest version of TripleA before playing this save.\n\n"
+          + "Do you wish to continue and open this save with your current 'old' version?";
```

Comment:
Looking forward to [Raw String Literals coming in java 12](http://openjdk.java.net/jeps/326) ^^